src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -45,6 +49,9 @@ using namespace chip;
 using namespace ::chip::DeviceLayer;
 using namespace ::chip::Transport;
 
+constexpr uint16_t kDACCertificate = 1;
+constexpr uint16_t kPAICertificate = 2;
+
 // As per specifications section 11.22.5.1. Constant RESP_MAX
 constexpr uint16_t kMaxRspLen = 900;
```

Comment:
This should be in the anonymous namespace.

Reply:
will update